lib/rucio/daemons/hermes/hermes.py

```diff
@@ -257,11 +257,14 @@ def deliver_messages(once=False, brokers_resolved=None, thread=0, bulk=1000, del
                                              heartbeat['nr_threads'],
                                              host_and_ports)
                                 conn.connect(wait=True)
+
                         conn.send(body=json.dumps({'event_type': str(message['event_type']).lower(),
                                                    'payload': message['payload'],
                                                    'created_at': str(message['created_at'])}),
                                   destination=destination,
-                                  headers={'persistent': 'true'})
+                                  headers={'persistent': 'true',
+                                           'event_type': str(message['event_type']).lower()})
```

Comment:
event_type right now is only email or message? Or am I wrong? Not sure if this is sufficient for the ticket? Wasn't the point to be able to be more selective?

Reply:
event_type is "transfer-done", etc...